kola/tests/misc/raid.go

```diff
@@ -80,6 +152,18 @@ func init() {
 		Name:        "coreos.disk.raid.root",
 	})
 	register.Register(&register.Test{
+		// This test needs additional disks which is only supported on qemu since Ignition
+		// does not support creating partitions on disks without wiping the partition table
+		// first.
+		// TODO(ajeddeloh): change this to create partitions 9 and 10 once Ignition supports
+		// creating them
+		Run:         NestedRootOnRaid,
+		ClusterSize: 0,
+		Platforms:   []string{"qemu"},
+		MinVersion:  semver.Version{Major: 1520},
```

Comment:
Will need to be updated, of course.

Reply:
1534 good? It's dependent on Ignition changes (which will require a ignition bump in the overlay).

Reply:
Seems like a reasonable guess.  If it turns out to be wrong, the test will start failing and we'll bump again.